embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceCacheService.kt

```diff
@@ -66,7 +65,7 @@ internal class EmbraceCacheService(
 
     private fun File.writeHalfTheBytes(array: ByteArray): Unit = FileOutputStream(this).use {
         it.write(array.dropLast(array.size / 2).toByteArray())
-        throw RuntimeException("yooooo")
+        throw IllegalAccessException("yooooo")
```

Comment:
i'm assuming we're only using this and the previous method in tests. should we at least mark them as such if there is no better alternative place for them to be?

Reply:
Oh this will be removed before I merge it. None of this stuff get into production code :-)